lib/vecmat.gi

```diff
@@ -1411,12 +1411,12 @@ local sf, rep, ind, ind2, row, i,big,l;
   else
     if not IsField(field) then
       # not a field
-      return Immutable(matrix);
+      return matrix;
```

Comment:
Why this change? It is not mentioned in your commit message, and at a quick glance, seems wrong.